### ISSUE TITLE
Object Identifier encoded wrong in DODF-bootstrap example

### ISSUE THREAD

**Issue by stig-bjorlykke** [OpenMobileAlliance/OMA_LwM2M_for_Developers#532]
The OMA-TS-LightweightM2M specifications (``V1.0.2``, ``V1.1.1`` and ``V1.2``) chapter ``G.3.3 Data Object Directory File, EF (DODF-bootstrap)`` describes a registered OID (``2.23.43.9.1``) and a DER encoded ASN.1 format for this record.

In the example provided the OID is encoded twice (Tag and Length is duplicated), and decoding this OID gives a wrong value (``0.6.4.103.43.9.1``). The specification states that the OidDO structure MUST contain an id value set to the registered OID, and this is not true in the example.

The attached [DODF-bootstrap.pcapng.zip](https://github.com/OpenMobileAlliance/OMA_LwM2M_for_Developers/files/7074621/DODF-bootstrap.pcapng.zip) contains a ``pcapng`` file with this example imported, which can be analysed using Wireshark. This is the textual output from the Wireshark decoder:
```
[CONTEXT 1]
    SEQUENCE
    SEQUENCE
        UTF8String: LwM2M Bootstrap
    [CONTEXT 1]
        SEQUENCE
            OID: 0.6.4.103.43.9.1 (itu-t.6.4.103.43.9.1)
            SEQUENCE
                OCTETSTRING: 6432
```

This example with the wrong OID has been found in live SIM cards.

**Comment by hannestschofenig** [comment]
Thanks for pointing this out. We will discuss it in one of our upcoming conference calls.

**Comment by mkgillmore** [comment]
Resolved in 1.2x release
